e2e/components/button.playwright-spec.ts

```diff
@@ -1,16 +1,14 @@
 import { expect, Locator, Page, test } from '@playwright/test';
 
-const component = {
-    stateAndStyle: 'dev-button-state-and-style'
-};
+const getThemeToggle = (page: Page) => page.getByTestId('e2eThemeToggle');
 
 test.describe('KbqButtonModule', () => {
-    test.describe(component.stateAndStyle, () => {
-        const goToPage = (page: Page) => page.goto('');
-        const getComponent = (page: Page) => page.locator(component.stateAndStyle);
-        const togglePrefix = (locator: Locator) => locator.getByText('show prefix icon').click();
-        const toggleTitle = (locator: Locator) => locator.getByText('show title').click();
-        const toggleSuffix = (locator: Locator) => locator.getByText('show suffix icon').click();
+    test.describe('DevButtonStateAndStyle', () => {
```

Comment:
можно использовать имя компонента

```suggestion
    test.describe(DevButtonStateAndStyle.name, () => {
```

Reply:
нет нельзя, это нужно дополнительно настраивать